lib/utils/constants.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+abstract class Constants {
+  static const double personNameFontSize = 45.0;
+  static const double phoneNumberFontSize = 20.0;
+  static const double positionFontSize = 20.0;
+  static const double personNameAndPositionSizeBoxHeight = 10.0;
+  static const double avatarAndPersonNameSizeBoxHeight = 45.0;
+  static const double phoneAndEmailSizeBoxHeight = 10.0;
+  static const double avatarRadius = 100.00;
+  static const double emailFontSize = 20.0;
+  static const double avatarBlurShadow = 35.0;
+
+  // contactInfoTextStyle
+  static const mailAndPhoneNumberFontSize = 20.0;
+  static const letterSpacing = 1.0;
+  static const contactInfoTextColor = Colors.white;
```

Comment:
This constant is not been used, you should remove it, also try not to make constants out of constants

Reply:
done